src/screens/Assets/collectibles/CollectiblesTab.js

```diff
@@ -50,9 +50,13 @@ import type { Chain } from 'models/Chain';
 // Local
 import { type FlagPerChain, useExpandItemsPerChain } from '../utils';
 import { type CollectibleItem, useCollectibleAssets } from './selectors';
-import { buildCollectibleFromCollectibleItem } from './utils';
+import { buildCollectibleFromCollectibleItem, calculateTotalCollectibleCount } from './utils';
 import CollectibleListItem from './CollectibleListItem';
 
+// assets (Temporarily removed banner for empty wallets until design change)
+// const collectibleBanner = require('assets/images/banners-collectibles.png');
```

Comment:
Same as above, please remove dead code

Reply:
this one is Temporarily removed for NFT banner